base/deprecated.jl

```diff
@@ -1429,6 +1366,16 @@ end
 using .DSP
 export conv, conv2, deconv, filt, filt!, xcorr
 
+module Test
+for f in [Symbol("@inferred"), Symbol("@test"), Symbol("@test_approx_eq"),
+          Symbol("@test_approx_eq_eps"), Symbol("@test_broken"), Symbol("@test_nowarn"),
+          Symbol("@test_skip"), Symbol("@test_throws"), Symbol("@test_warn"),
+          Symbol("@testset"), :GenericArray, :GenericDict, :GenericSet, :GenericString,
+          :detect_ambiguities, :detect_unbound_args]
+    @eval Base.@deprecate_moved $f "Test" true true
+end
+end
```

Comment:
Can we do `Base.@deprecate_moved Test "Test" true true`, or would that do something different?

Reply:
`@deprecate_moved` only seems to support the format `@deprecate_moved function package`.